doc/nrf/known-warnings.txt

```diff
@@ -1,5 +1,5 @@
 # Each line should contain the regular expression of a known Sphinx warning
 # that should be filtered out
-.*Duplicate C declaration.*\n.*'\.\. c:function:: .*bt_mesh_light_ctl_set.*'.*
-.*Duplicate C declaration.*\n.*'\.\. c:function:: .*bt_mesh_light_temp_set.*'.*
-.*Duplicate C declaration.*\n.*'\.\. c:struct:: bt_dfu_smp_rsp_state'.*
+.*Duplicate C declaration.*\n.*'\.\. c:.*:: .*bt_mesh_light_ctl_set.*'.*
```

Comment:
Any plans to root-cause these and see if we're masking real problems at some point?